pipelines/phenology_EO.json

```diff
@@ -0,0 +1,269 @@
+    },
+    {
+      "source": "3",
+      "sourceHandle": "timeseries",
+      "target": "5",
+      "targetHandle": null,
+      "id": "reactflow__edge-3timeseries-5"
+    },
+    {
+      "source": "3",
+      "sourceHandle": "rasters",
+      "target": "6",
+      "targetHandle": "rasters",
+      "id": "reactflow__edge-3rasters-6rasters"
+    },
+    {
+      "source": "7",
+      "sourceHandle": null,
+      "target": "3",
+      "targetHandle": "start_date",
+      "id": "reactflow__edge-7-3start_date"
+    },
+    {
+      "source": "8",
+      "sourceHandle": null,
+      "target": "3",
+      "targetHandle": "end_date",
+      "id": "reactflow__edge-8-3end_date"
+    },
+    {
+      "source": "8",
+      "sourceHandle": null,
+      "target": "6",
+      "targetHandle": "end_date",
+      "id": "reactflow__edge-8-6end_date"
+    },
+    {
+      "source": "7",
+      "sourceHandle": null,
+      "target": "6",
+      "targetHandle": "start_date",
+      "id": "reactflow__edge-7-6start_date"
+    },
+    {
+      "source": "6",
+      "sourceHandle": "phenology_change",
+      "target": "9",
+      "targetHandle": null,
+      "id": "reactflow__edge-6phenology_change-9"
+    }
+  ],
+  "inputs": {
+    "phenology>summarise_phenology.yml@3|bands": {
+      "description": "Bands of interest for the calculations",
+      "label": "Bands",
+      "type": "options[]",
+      "options": [
+        "SOSD",
+        "EOSD",
+        "SOSV",
+        "EOSV",
+        "MAXD",
+        "MAXV",
+        "MINV",
+        "AMPL",
+        "LENGTH",
+        "LSLOPE",
+        "RSLOPE",
+        "SPROD",
+        "TPROD"
+      ],
+      "example": [
+        "LENGTH",
+        "AMPL"
+      ],
+      "weight": 0
+    },
+    "phenology>summarise_phenology.yml@3|aggregate_function": {
+      "description": "Function to spatially aggregate by",
+      "label": "Aggregate function",
+      "type": "options",
+      "options": [
+        "mean",
+        "min",
+        "max"
+      ],
+      "example": "mean",
+      "weight": 1
+    },
+    "phenology>summarise_phenology.yml@3|spatial_resolution": {
+      "description": "Spatial resolution of the raster for plotting, leave blank to have the original spatial resolution of the layer.",
+      "label": "Spatial resolution",
+      "type": "int",
+      "weight": 2,
+      "example": null
+    },
+    "data>getBoundingBox.yml@1|country": {
+      "description": "Country of interest. For list of countries look here https://geoio.biodiversite-quebec.ca/docs#/default/validate_country_name_validate_country_name_get.",
+      "label": "Country",
+      "type": "text",
+      "example": "Colombia",
+      "weight": 3
+    },
+    "data>getBoundingBox.yml@1|state": {
+      "description": "State or province of interest (leave blank if you want the full country). For list of accepted region names look here https://geoio.biodiversite-quebec.ca/docs#/default/validate_state_name_validate_state_name_get.",
```

Comment:
THese should be markdown links. If they are not in the original script inside the pipeline, please edit the corresponding script. Same above. If they have been changed in the script, we'll need to paste it here... that's something I'd like to make automatic one day...